src/example3.cpp

```diff
@@ -81,7 +81,7 @@ int main(int /* argc */, char ** /* argv */) {
 
     int width, height;
     glfwGetFramebufferSize(window, &width, &height);
-    glViewport(0, 0, width, height);
```

Comment:
This is incorrect: the point of these lines is to clear the screen once before the mainloop. Application startup often takes a second or so (e.g. to load textures and other resources), and annoying garbage can be seen in the meantime on some platforms without this step.